data/hai/hai reveal 4 middle.txt

```diff
@@ -97,37 +96,37 @@ mission "Hai Reveal [A02] Symbol Identification"
+			`	"The Geodesic Geocoris you found has self-repair systems that use technology which could be responsible for something like this, and we suspect they came from the same place."`
 				goto questions
 			label patterns
-			`	Teeneep explains, "I head a large activist organization for social direction and civil rights alignment; we have some resources of our own. We've combed police records for the past few decades and noticed patterns in disappearances."`
-			`	"There are violent crimes and kidnappings in Hai space, though much less frequently than in human space. Murder, assault, estranged parents kidnapping their children, all of that happens here. Many of those crimes are solved, eventually: the victim is found dead or alive. Some of them are not solved. There are also disappearances that go unreported until long after they occur. Hai who simply get up and decide to go on an adventure one day out of boredom, and then get stuck up a mountain, fall into a crevasse, or any encounter any number of other fatal natural phenomena. Their bodies are sometimes found, but they're usually only hard to find when they're local to the destination with the danger."`
-			`	"In recent years a disproportionate number are from the intelligentsia: doctors, scientists, engineers, architects. In particular, they are specialists of some kind in their field, but not prominent enough for the field to be significantly set back by their absence. The same is happening in human space, people with niche skills are occasionally disappearing. In Hai space particularly, some of these disappearances are from worlds where we would expect to be able to track them to the general area of their disappearance, but, instead, they are simply... gone. We believe someone is targeting researchers and specialists and trying to hide it beneath the noise."`
+			`	"First, you must understand that violent crimes in Hai space are not a rare thing, though they aren't a day-to-day part of life as in human space. Murder, assault, estranged parents kidnapping their children, all of that happens here. Some Hai go on an adventure one day and then get fall in a crevasse or get eaten by a beast. Some join the Unfettered, but don't tell anyone. Most cases are solved: the person is found dead or alive. Sometimes, though, the no person or body is ever found."`
+			`	"In recent years, a disproportionate number of Hai and human kidnapping victims seem to be purposefully picked from the intelligentsia: doctors, scientists, engineers, architects. Most were specialists within their field. None were prominent enough for their departure to send shock waves in the community though. Some disappeared from worlds where we'd expect to be able to track them to their general area, but instead, they are simply... gone.`
```

Comment:
I think this is overall an improvement but feel it's still pretty awkward.

I think a fair bit of it comes from the switch to directly calling these kidnapping victims, but I also think that switch is appropriate for assumptions the speaker would be making. I think rectifying the awkwardness further will require either walking that back or more clearly setting it in stone.

Reply:
Yes, transitioning from "disappearances" at the top of the dialog to "probable kidnappings" at the bottom seems better. I'll ponder this and update the text later.

Reply:
I have updated the whole conversation, changing "kidnapping" to "disappearance" in most places. "Kidnapping" is only mentioned as a suspected crime, for known victims, or as a reason for precautions.